### PR TITLE
adding terragrunt-read-config after hook

### PR DESCRIPTION
**Pull request by sleungcy**
#1020

### CODE REVIEW

**Comment by yorinasub17** [comment]
Looks like the build failed with:

```
--- FAIL: TestTerragruntBeforeAndAfterHook (8.34s)
    integration_test.go:2269: Copying fixture-hooks/before-and-after to /tmp/terragrunt-test615422508
    integration_test.go:358: 
        	Error Trace:	integration_test.go:358
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 2
        	Test:       	TestTerragruntBeforeAndAfterHook
        	Messages:   	Hooks on terragrunt-read-config command executed more than once
```

**Comment by sleungcy** [comment]
I fixed the test

**Comment by sleungcy** [comment]
@yorinasub17 Could you help rerun the ci please?

File: test/integration_test.go [hunk 1]
```diff
@@ -341,11 +341,23 @@ func TestTerragruntBeforeAndAfterHook(t *testing.T) {
...
 
+	output := stderr.String()
+
+	if err != nil {
+		t.Errorf("Did not expect to get error: %s", err.Error())
+	}
+
+	assert.Equal(t, 0, strings.Count(output, "BEFORE_TERRAGRUNT_READ_CONFIG"), "terragrunt-read-config before_hook should not be triggered")
+	t.Logf("output: %s", output)
+	assert.Equal(t, 2, strings.Count(output, "AFTER_TERRAGRUNT_READ_CONFIG"), "Hooks on terragrunt-read-config command executed more than once")
```
Comment (yorinasub17) [inline test/integration_test.go]: NIT: can you add a comment indicating that it is twice because terragrunt echos the command it is running?

**Comment by yorinasub17** [comment]
Thanks for adding that comment! Since the last build passed, I'm going to go ahead and merge + release this. Thanks for your contribution!

**Comment by yorinasub17** [comment]
https://github.com/gruntwork-io/terragrunt/releases/tag/v0.21.12

Binaries will show up shortly (~10 mins).

**Pull request merged**